x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/index.tsx

```diff
@@ -178,11 +178,17 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
     });
   }, [search, selectedAgentPolicies, selectedStatus, selectedTags]);
 
+  // filters kuery for all tags
+  const kueryAllTags = useMemo(() => {
+    return getKuery({});
+  }, []);
+
   const [agentsOnCurrentPage, setAgentsOnCurrentPage] = useState<Agent[]>([]);
   const [agentsStatus, setAgentsStatus] = useState<
     { [key in SimplifiedAgentStatus]: number } | undefined
   >();
   const [allTags, setAllTags] = useState<string[]>();
+  const [allAddRemoveTags, setAllAddRemoveTags] = useState<string[]>();
```

Comment:
Naming nit: should this be called `allTags` and the existing `allTags` renamed to  e.g.`filteredTags`?